docs/input-variables.md

```diff
@@ -21,6 +21,7 @@ name                                | default                 | description
 etcdShape                           | VM.Standard1.1          | OCI shape for etcd nodes
 k8sMasterShape                      | VM.Standard1.1          | OCI shape for k8s master(s)
 k8sWorkerShape                      | VM.Standard1.2          | OCI shape for k8s worker(s)
+master_oci_lb_enabled               | true                    | enable/disable the k8s master oci load balancer. true: use oci load balancer for k8s master . false: use nginx reverse proxy as a software load balancer for k8s masters.
```

Comment:
"false: use nginx reverse proxy as a software load balancer for k8s masters" => "false: use software-based reverse proxy as a software load balancer for k8s masters"